contracts/smartdeploy/src/registry/contract.rs

```diff
@@ -126,12 +139,62 @@ impl IsDeployable for ContractRegistry {
             .take(limit.unwrap_or_else(|| self.0.len()) as usize);
         let mut res = vec![env()];
         for item in items {
-            res.push_back(item);
+            res.push_back((item.0, item.1.contract_id().clone()));
```

Comment:
Is this clone necessary?

Reply:
Yes because the expected return type is `Vec<(soroban_sdk::String, soroban_sdk::Address)`.
`item.1.contract_id()` returns `&Address`. I used `clone()` to convert `&Address` into `Address`